hsclient/hydroshare.py

```diff
@@ -618,6 +1100,10 @@ def file_aggregate(self, path: str, agg_type: AggregationType, refresh: bool = T
         if refresh:
             # Only return the newly created aggregation if a refresh is requested
             self.refresh()
+            if agg_type == AggregationType.GeographicRasterAggregation and not path.endswith(".vrt") \
+                    or agg_type == AggregationType.FileSetAggregation:
+                # search all files of the aggregation to find a matching aggregation
+                return self.aggregation(file__path=path, file_path_priority=False)
```

Comment:
This is a clue to why the block above is needed.  I don't understand yet why we need the file_path_priority filter and why rasters and filesets are treated differently than other aggregations.

Reply:
I have removed these changes as I am no more doing the priority aggregation path-based search.